pom.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.6.6</version>
+    <relativePath></relativePath>
+    <!-- lookup parent from repository -->
+  </parent>
   <groupId>com.semanticbits.terminologyservice</groupId>
```

Comment:
Will not make any difference but I think `gov.cms.madie` would be good

Reply:
Thanks Ashok! gov.cms.madie  or cms.gov.madie?

Reply:
gov.cms.madie

Reply:
Ok thanks Greg!

Reply:
groupId is changed.